packages/web-ui/src/Grid/Grid.tsx

```diff
@@ -1,7 +1,8 @@
-import MuiGrid from '@mui/material/Grid';
+import MuiGrid from '@mui/system/Unstable_Grid';
 import type { OverridableComponent, OverrideProps } from '@mui/material/OverridableComponent';
 import { forwardRef } from 'react';
-import type { GridProps as MuiGridProps, RegularBreakpoints } from '@mui/material/Grid';
+import type { GridProps as MuiGridProps } from '@mui/system/Unstable_Grid';
+import { RegularBreakpoints } from '@mui/material/Grid';
```

Comment:
This is only available from @mui/material

Reply:
we can just copy paste from the material source then.....

```tsx
export type RegularBreakpoint = boolean | 'auto' | number

export interface CustomGridProps {
   mobile?: RegularBreakpoint;
   tablet?: RegularBreakpoint;
   desktop?: RegularBreakpoint;
   wide?: RegularBreakpoint;
 }
```